### ISSUE TITLE
ShortCode is not working.

### ISSUE THREAD

**Issue by kishoresahoo** [TheLookandFeel/listings#71]
[listings] shortcode is not working. I am not sure it is yet to implement.

**Comment by scottbasgaard** [comment]
Hey @kishoresahoo,

This "should" work as of now and can confirm on 0.2.0 on trunk here on GitHub.

Would love to figure out what is causing this for you and appreciate your input / feedback as well!

If you visit the listings archive page do they show?
- http://your.site/listings/

Will do some more playing around tomorrow to see if something is up and don't want to start pinpointing possible conflicts just yet.

If you have any screenshots / links you can provide of your example listings that should show, the edit page where you are using the shortcode and the front-end where this should render that could be helpful as well.

Cheers,
Scott

cc: @coenjacobs

**Comment by kishoresahoo** [comment]
Here I am attaching snap. @scottbasgaard @coenjacobs . I am using 0.2.0 .

This page http://your.site/listings/ is breaking. Title is not coming.also load more text.

![listings wordpress](https://cloud.githubusercontent.com/assets/5745798/17048289/6d8526f2-5002-11e6-9bf3-239d0d319ab4.png)
![test wordpress](https://cloud.githubusercontent.com/assets/5745798/17048298/8403d626-5002-11e6-812f-674f0011e5cc.png)

**Comment by kishoresahoo** [comment]
I think need to make ajax call. Code present in ajax-filters.js in WPJM.

**Comment by scottbasgaard** [comment]
@kishoresahoo think @coenjacobs is aware of the issue and we'll have a fix out soon.

Thanks again for the report.

**Comment by scottbasgaard** [comment]
Related:
- https://wordpress.org/support/topic/display-listings-wo-doing-a-search?replies=4#post-8754144

**Comment by scottbasgaard** [comment]
@coenjacobs looks like we need to port over frontend CSS for generic listings.

What are your thoughts here.

1. Move div.job_listings over to core and do something generic like div.listings for this which we use in all extensions.

2. Duplicate div.job_listings styling into core and each extension.

**Comment by evertsemeijn** [comment]
@scottbasgaard I agree with your port of _.listings_. Regarding 2. you could inject an extra extension-based class so you can reuse the _.listings_ declarations and overwrite/extend with _.listings.jobs_.

Does that make sense?

cc @coenjacobs
